modules/tinymce/src/themes/silver/main/ts/ui/header/InlineHeader.ts

```diff
@@ -148,22 +163,13 @@ export const InlineHeader = (
             ? Math.max(targetBounds.y - Height.get(container.element) + offset, 0)
             : targetBounds.y - offsetBox.y + scrollDelta - Height.get(container.element) + offset;
 
-          return {
-            top: isPositionedAtTop()
-              ? topValue
-              : targetBounds.bottom,
-            left: isOffsetParentBody
-              ? targetBounds.x
-              : targetBounds.x - offsetBox.x
-          };
+          return isPositionedAtTop()
+            ? topValue
+            : targetBounds.bottom;
         }
       );
 
-      const baseProperties = {
-        position: 'absolute',
-        left: Math.round(left) + 'px',
-        top: Math.round(top) + 'px'
-      };
+      const left = getLeft();
```

Comment:
Can this be inlined to the `baseProperties` object below, as `getTop` was? I can't see where else it's used.

Reply:
`left` is also used at line 189 to calculate the `availableWidth`: https://github.com/tinymce/tinymce/pull/9305/files#diff-5319872f9b33029984fecd3a8a0965ef53643e92cd5ba4d846cbd98f23dbd523L183